app/models/shift.rb

```diff
@@ -31,21 +32,33 @@ def can_be_dropped?
   end
 
   private
+  def any_shift_time_blank?
+    return if self.shift_start.blank? || self.shift_end.blank?
+  end
 
   def shift_not_too_short
-    return if self.shift_start.blank? || self.shift_end.blank?
+    any_shift_time_blank?
```

Comment:
Then here, you call the return if:

```suggestion
    return if any_shift_time_blank?
```